From 1c12e96e6547ec459fabaa8467621fa622285e8e Mon Sep 17 00:00:00 2001
From: Atman <masteratman@gmail.com>
Date: Tue, 4 Jun 2019 10:28:24 +0000
Subject: [PATCH 10/11] Revert "sepol: Label vendor.qcom.bluetooth.soc"

This will become unused after we start reading the vendor.bluetooth.soc prop, so we drop it.

This reverts commit 4edd29ff2a2cd19326fc557403f6ba0e6847744f.

Change-Id: I4aacf1b91148ae49561e0ee47c11bcdafe6dc202
Signed-off-by: Atman <masteratman@gmail.com>
---
 qcom/common/property_contexts | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 qcom/common/property_contexts

diff --git a/qcom/common/property_contexts b/qcom/common/property_contexts
deleted file mode 100644
index 15cdf0f..0000000
--- a/qcom/common/property_contexts
+++ /dev/null
@@ -1 +0,0 @@
-vendor.qcom.bluetooth.     u:object_r:exported3_default_prop:s0
-- 
2.17.1

